src/DB/EntityRepository/User/Notification/NotificationRepository.php

```diff
@@ -76,4 +76,15 @@ public function getFollowNotificationForUser(User $owner = null, User $follow_fr
 
     return $qb->getQuery()->getResult();
   }
+
+  public function getNotificationByID(int $id): ?CatroNotification
+  {
+    $query_builder = $this->createQueryBuilder('e');
```

Comment:
Nothing wrong with the query. Just FYI every entity repository should already have a find method which searches an entity by its ID